src/cmd-build

```diff
@@ -236,36 +236,36 @@ echo "New build ID: ${buildid}"
 imageprefix="${name}"-"${buildid}"
 # Make these two verbose
 set -x
-mkdir -p tmp/anaconda
-img_base=tmp/${imageprefix}-base.qcow2
-
-# forgive me for this sin
-checksum_location=$(find /usr/lib/coreos-assembler-anaconda/ -name '*CHECKSUM' | head -1)
 
 build_image() {
 	local size kargs
 	size="$(python3 -c 'import sys, yaml; print(yaml.load(sys.stdin, Loader=yaml.CLoader)["size"])' < "$configdir/image.yaml")G"
 	kargs="$(python3 -c 'import sys, yaml; args = yaml.load(sys.stdin, Loader=yaml.CLoader).get("extra-kargs", []); print(" ".join(args))' < "$configdir/image.yaml")"
-	kargs="$kargs console=tty0 console=${VM_TERMINAL},115200n8"
+	kargs="$kargs console=tty0 console=${VM_TERMINAL},115200n8 ignition.platform.id=qemu"
 
-	qemu-img create -f qcow2 "$img_base" "$size"
-	runvm_with_disk "$img_base" /usr/lib/coreos-assembler/create_disk.sh /dev/vda "$tmprepo" "${ref-:${commit}}" /usr/lib/coreos-assembler/grub.cfg "$name" "\"$kargs\""
+	qemu-img create -f qcow2 "$1" "$size"
+	runvm_with_disk "$1" /usr/lib/coreos-assembler/create_disk.sh /dev/vda "$tmprepo" "${ref-:${commit}}" /usr/lib/coreos-assembler/grub.cfg "$name" "\"$kargs\""
 }
 
 if [ -n "${build_qemu}" ]; then
     img_qemu=${imageprefix}-qemu.qcow2
     if [ "$arch" == "x86_64" ]; then
-        build_image
+        build_image "$img_qemu"
+        vm_iso_checksum="n/a"
```

Comment:
Even better: can we can drop this completely from `meta.json` in the x86_64 path since we're not using a "VM ISO" anymore?